.ci/releasenotes-gen-xdoc-push.sh

```diff
@@ -51,7 +51,7 @@ java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.
      -githubAuthToken "$READ_ONLY_TOKEN" \
      -generateXdoc \
      -xdocTemplate $BUILDER_RESOURCE_DIR/templates/xdoc_freemarker.template \
-     -publishXdoc -publishXdocWithPush
+     -publishXdoc
```

Comment:
Ok, but please do commit by shell commands

Reply:
This script must all work.

I presume `publishXdoc** - (optional) Make commit in local checkstyle repo with releasenotes.` is place where there i problem, so better to do this by shell commands, that very robust.